integration-cli/docker_cli_events_test.go

```diff
@@ -65,9 +66,29 @@ func (s *DockerSuite) TestEventsContainerFailStartDie(c *check.C) {
 }
 
 func (s *DockerSuite) TestEventsLimit(c *check.C) {
-	for i := 0; i < 30; i++ {
-		dockerCmd(c, "run", "busybox", "echo", strconv.Itoa(i))
+
+	var waitGroup sync.WaitGroup
+	errors := make(chan error, 17)
```

Comment:
nit, I'd call it `errChan` since errors could be imported (again nit, don't necessary do it and maintainers have last word of course)